controllers/metallb_controller.go

```diff
@@ -82,7 +103,7 @@ func (r *MetallbReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
-func (r *MetallbReconciler) syncMetalLBResources(config *metallbv1alpha1.Metallb) error {
+func (r *MetallbReconciler) syncMetalLBResources(config *metallbv1alpha1.Metallb) (bool, error) {
```

Comment:
Add a godoc comment saying what's the meaning of the returned flag

Reply:
Flag removed